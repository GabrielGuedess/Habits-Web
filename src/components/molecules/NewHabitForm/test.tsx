import { render } from '@testing-library/react';

import { NewHabitForm } from '.';

describe('<NewHabitForm />', () => {
  it('should render the heading', () => {
    const { container } = render(<NewHabitForm />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
