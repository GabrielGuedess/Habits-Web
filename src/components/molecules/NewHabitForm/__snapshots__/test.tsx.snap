// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewHabitForm /> should render the NewHabitForm 1`] = `
<form
  class="mt-6 flex w-full flex-col"
>
  <label
    class="font-semibold leading-tight"
    for="title"
  >
    Qual seu comprometimento?
  </label>
  <input
    class="mt-3 rounded-lg bg-zinc-800 p-4 text-white placeholder:text-zinc-400"
    id="title"
    placeholder="Exercícios, dormir bem, etc..."
    type="text"
    value=""
  />
  <label
    class="mt-4 font-semibold leading-tight"
    for=""
  >
    Qual a recorrência?
  </label>
  <div
    class="mt-3 flex flex-col gap-2"
  >
    <button
      aria-checked="false"
      class="group flex items-center gap-3"
      data-state="unchecked"
      role="checkbox"
      type="button"
      value="on"
    >
      <div
        class="flex h-8 w-8 items-center justify-center rounded-lg border-2 border-zinc-800 bg-zinc-900 group-data-[state=checked]:border-green-500 group-data-[state=checked]:bg-green-500"
      />
      <span
        class="leading-tight text-white"
      >
        Domingo
      </span>
    </button>
    <input
      aria-hidden="true"
      style="transform: translateX(-100%); position: absolute; pointer-events: none; opacity: 0; margin: 0px; width: 0px; height: 0px;"
      tabindex="-1"
      type="checkbox"
      value="on"
    />
    <button
      aria-checked="false"
      class="group flex items-center gap-3"
      data-state="unchecked"
      role="checkbox"
      type="button"
      value="on"
    >
      <div
        class="flex h-8 w-8 items-center justify-center rounded-lg border-2 border-zinc-800 bg-zinc-900 group-data-[state=checked]:border-green-500 group-data-[state=checked]:bg-green-500"
      />
      <span
        class="leading-tight text-white"
      >
        Segunda-feira
      </span>
    </button>
    <input
      aria-hidden="true"
      style="transform: translateX(-100%); position: absolute; pointer-events: none; opacity: 0; margin: 0px; width: 0px; height: 0px;"
      tabindex="-1"
      type="checkbox"
      value="on"
    />
    <button
      aria-checked="false"
      class="group flex items-center gap-3"
      data-state="unchecked"
      role="checkbox"
      type="button"
      value="on"
    >
      <div
        class="flex h-8 w-8 items-center justify-center rounded-lg border-2 border-zinc-800 bg-zinc-900 group-data-[state=checked]:border-green-500 group-data-[state=checked]:bg-green-500"
      />
      <span
        class="leading-tight text-white"
      >
        Terça-feira
      </span>
    </button>
    <input
      aria-hidden="true"
      style="transform: translateX(-100%); position: absolute; pointer-events: none; opacity: 0; margin: 0px; width: 0px; height: 0px;"
      tabindex="-1"
      type="checkbox"
      value="on"
    />
    <button
      aria-checked="false"
      class="group flex items-center gap-3"
      data-state="unchecked"
      role="checkbox"
      type="button"
      value="on"
    >
      <div
        class="flex h-8 w-8 items-center justify-center rounded-lg border-2 border-zinc-800 bg-zinc-900 group-data-[state=checked]:border-green-500 group-data-[state=checked]:bg-green-500"
      />
      <span
        class="leading-tight text-white"
      >
        Quarta-feira
      </span>
    </button>
    <input
      aria-hidden="true"
      style="transform: translateX(-100%); position: absolute; pointer-events: none; opacity: 0; margin: 0px; width: 0px; height: 0px;"
      tabindex="-1"
      type="checkbox"
      value="on"
    />
    <button
      aria-checked="false"
      class="group flex items-center gap-3"
      data-state="unchecked"
      role="checkbox"
      type="button"
      value="on"
    >
      <div
        class="flex h-8 w-8 items-center justify-center rounded-lg border-2 border-zinc-800 bg-zinc-900 group-data-[state=checked]:border-green-500 group-data-[state=checked]:bg-green-500"
      />
      <span
        class="leading-tight text-white"
      >
        Quinta-feira
      </span>
    </button>
    <input
      aria-hidden="true"
      style="transform: translateX(-100%); position: absolute; pointer-events: none; opacity: 0; margin: 0px; width: 0px; height: 0px;"
      tabindex="-1"
      type="checkbox"
      value="on"
    />
    <button
      aria-checked="false"
      class="group flex items-center gap-3"
      data-state="unchecked"
      role="checkbox"
      type="button"
      value="on"
    >
      <div
        class="flex h-8 w-8 items-center justify-center rounded-lg border-2 border-zinc-800 bg-zinc-900 group-data-[state=checked]:border-green-500 group-data-[state=checked]:bg-green-500"
      />
      <span
        class="leading-tight text-white"
      >
        Sexta-feira
      </span>
    </button>
    <input
      aria-hidden="true"
      style="transform: translateX(-100%); position: absolute; pointer-events: none; opacity: 0; margin: 0px; width: 0px; height: 0px;"
      tabindex="-1"
      type="checkbox"
      value="on"
    />
    <button
      aria-checked="false"
      class="group flex items-center gap-3"
      data-state="unchecked"
      role="checkbox"
      type="button"
      value="on"
    >
      <div
        class="flex h-8 w-8 items-center justify-center rounded-lg border-2 border-zinc-800 bg-zinc-900 group-data-[state=checked]:border-green-500 group-data-[state=checked]:bg-green-500"
      />
      <span
        class="leading-tight text-white"
      >
        Sábado
      </span>
    </button>
    <input
      aria-hidden="true"
      style="transform: translateX(-100%); position: absolute; pointer-events: none; opacity: 0; margin: 0px; width: 0px; height: 0px;"
      tabindex="-1"
      type="checkbox"
      value="on"
    />
  </div>
  <button
    class="mt-6 flex items-center justify-center gap-3 rounded-lg bg-green-600 p-4 font-semibold transition-colors hover:bg-green-500"
    type="submit"
  >
    <svg
      fill="currentColor"
      height="20"
      viewBox="0 0 256 256"
      width="20"
      xmlns="http://www.w3.org/2000/svg"
    >
      <rect
        fill="none"
        height="256"
        width="256"
      />
      <polyline
        fill="none"
        points="216 72 104 184 48 128"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="24"
      />
    </svg>
     Confirmar
  </button>
</form>
`;
